L1Trigger/L1TTwinMux/src/DTRPCBxCorrection.cc

```diff
@@ -21,17 +21,18 @@ using namespace std;
 DTRPCBxCorrection::DTRPCBxCorrection(L1MuDTChambPhContainer inphiDTDigis, L1MuDTChambPhContainer inphiRPCDigis)  {
       // const std::vector<L1MuDTChambPhDigi>* vInCon = i.getContainer();
 
-	m_phiDTDigis.setContainer(*inphiDTDigis.getContainer());
-	m_phiRPCDigis.setContainer(*inphiRPCDigis.getContainer());
-       
+//	m_phiDTDigis.setContainer(*inphiDTDigis.getContainer());
+//	m_phiRPCDigis.setContainer(*inphiRPCDigis.getContainer());
+  m_phiDTDigis=inphiDTDigis;
+  m_phiRPCDigis=inphiRPCDigis;
```

Comment:
You could initialise the members in the same way as for the DTLowQMatching class.